sklearn/ensemble/forest.py

```diff
@@ -468,6 +533,12 @@ def _set_oob_score(self, X, y):
     def _validate_y_class_weight(self, y):
         check_classification_targets(y)
 
+        # use this local class_weight to easily handle
```

Comment:
Not really sure what this comment means. This is a bit tricky to understand what is happening with class_weight and self.class_weight. It might be good to clarify this part of the code.